tests/test_repository.py

```diff
@@ -215,14 +300,12 @@ def dictfunc():
         ([], utils.DEFAULT_REPOSITORY, set(),),
     ],
 )
-def test_release_urls(package_meta, repository_url, release_urls):
+def test_release_urls(package_meta, repository_url, release_urls, repo):
+    """Test that the correct release urls are read"""
     packages = [
         pretend.stub(safe_name=name, metadata=pretend.stub(version=version),)
         for name, version in package_meta
     ]
 
-    repo = repository.Repository(
-        repository_url=repository_url, username="username", password="password",
-    )
-
+    repo.url = repository_url
```

Comment:
I would revert this change, and not use the `repo` fixture in this test; I think it makes the test setup less clear.

Reply:
Fixed in latest commit